snapcraft/internal/build_providers/_lxd/_images.py

```diff
@@ -30,7 +30,7 @@ def get_image_source(*, base: str) -> Dict[str, str]:
     return dict(
         type="image",
         mode="pull",
-        server="https://cloud-images.ubuntu.com/minimal/releases/",
+        server="https://cloud-images.ubuntu.com/buildd/daily",
```

Comment:
would this affect any images today (negatively)?

Reply:
Yes, they will have missing packages in some cases, but will match closely what is on launchpad. This should be fine considering that we had the "preview" message when using "--use-lxd".

Reply:
roger